vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -769,4 +769,8 @@ void LedgerImpl::Shutdown(ledger::ResultCallback callback) {
   });
 }
 
+void LedgerImpl::GetEventLogs(ledger::GetEventLogsCallback callback) {
+  database()->GetAllEventLogs(callback);
```

Comment:
Is there any danger of returning too much data here?

Reply:
limited to 2000 records